.github/workflows/ci.yml

```diff
@@ -49,6 +38,9 @@ jobs:
           bundler-cache: true
         env:
           BUNDLE_RUBYGEMS__PKG__GITHUB__COM: ${{ secrets.GITHUB_TOKEN }}
+      - run: |
+          docker run --name my_scylla_container -d -p 9042:9042 scylladb/scylla:${{ matrix.scylladb }}
+          sleep 100
```

Comment:
Would this be better run as a service? or did it not work that way for some reason? (I am assuming putting it under services would eliminate the need to sleep for 100 seconds) 

If it has to be done this way, does it really need 100s to start up? Just looking at the runtimes -- they seem a lot longer. However, if the plan is to just axe this repo when we migrate off scylla, then I'm good with this

Reply:
Didn't try it as a service on this version, but newer versions were able to start up successfully just before triggering a timeout that I couldn't seem to adjust. This pattern came from another repo. I can try tweaking the sleep some, but like you suggest, this is just around until we are off of scylla.